docs/docs/whitepaper.md

```diff
@@ -3,4 +3,24 @@ id: whitepaper
 title: Whitepaper
 ---
 
-Download the ZeppelinOS whitepaper [here](https://zeppelinos.org/zeppelin_os_whitepaper.pdf).
\ No newline at end of file
+Anyone can use open networks like Ethereum to run software and formalize
+contractual relationships through code. These decentralized applications often
+facilitate some kind of financial exchange for services. However, the
+development process to build this kind of software is often complex,
+time-consuming, and prone to error.
+
+Ensuring the security of these financial transactions and the funds held in
+smart contracts is imperative to the success of the blockchain ecosystem. There
+is currently little standardization and no easy way to develop highly secure
+applications. Responding effectively to security emergencies is beyond
+challenging with current tools.
+
+On the other hand, decentralized networks are not structured to work together,
+often needing access to each other’s native tokens to operate, making it hard
+to leverage external systems.
+
+To address these issues we propose zeppelin_os: an open-source, decentralized
```

Comment:
zeppelin_os -> ZeppelinOS

Reply:
(Update: it's still unresolved which we will use, but let's change this anyway and change back all if necessary).

Reply:
Fixed, and https://github.com/zeppelinos/whitepaper/pull/1